changelog/changelog.py

```diff
@@ -220,6 +220,11 @@ def get_parser():
                           help="Update an existing changelog file by prepending to it.",
                           )
 
+    optional.add_argument("--labels",
+                          nargs="+",
+                          help="Labels to update to the changelog",
```

Comment:
Nitpick: Slight clarification about how the labels are used

```suggestion
                          help="Labels to use for grouping PRs by category (the specified labels will be used "
                               "as headers in the changelog)",
```